app/helpers/partners_helper.rb

```diff
@@ -1,11 +1,10 @@
 # frozen_string_literal: true
 
 module PartnersHelper
-  def options_for_partners
```

Comment:
Oh, can you just check that this isn't breaking the users form on the offchance? (Reason: only definition of options_for_partners is here, only reference is in `views/admin/users/_form.html.erb`)